### PR TITLE
Update sso-saml-adfs-msws2016.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Added SAML metadata content to "Configure" section as per: mattermost/mattermost-webapp#4427 and https://mattermost.atlassian.net/browse/MM-21843.

### CODE REVIEW

File: source/deployment/sso-saml-adfs-msws2016.rst [hunk 1]
```diff
@@ -165,6 +165,10 @@ Configure SAML Sign-in for Mattermost
 
 	.. image:: ../../source/images/SSO-SAML-ADFS_configure-saml_001.png
 
+Alternatively, you can create a metadata URL by appending "FederationMetadata/2007-06/FederationMetadata.xml" to the root URL of the ADFS server, for example: ``https://<adfs.domain.com>/federationmetadata/2007-06/FederationMetadata.xml>``. 
+
+Then paste this URL in the **Identity Provider Metadata URL** field, then select **Get SAML Metadata from IdP**.
```
Comment (amyblais) [inline source/deployment/sso-saml-adfs-msws2016.rst]:
```suggestion
Then paste this URL in the **Identity Provider Metadata URL** field, and then select **Get SAML Metadata from IdP**.
```

Comment (catalintomai) [inline source/deployment/sso-saml-adfs-msws2016.rst]: Same comment as for https://github.com/mattermost/docs/pull/3341 : "Maybe add another phrase that gives the rationale for the alternate way of entering the metadata Url and clicking the button - something along the lines of: "clicking this button populates the SAML SSO URL and Identity Provider Issue URL and downloads the Idp certificate from the ADFS server - you do not have to populate these fields manually anymore.".

File: source/deployment/sso-saml-adfs-msws2016.rst [hunk 2]
```diff
@@ -155,7 +155,13 @@ You’re now about to finish configuring SAML for Mattermost!
...
 --------------------------------------
 
-1. Start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**.
+Create a metadata URL by appending "FederationMetadata/2007-06/FederationMetadata.xml" to the root URL of the ADFS server, for example: ``https://<adfs.domain.com>/federationmetadata/2007-06/FederationMetadata.xml>``. 
+
+Next, start Mattermost server and sign into Mattermost as a System Administrator. Go to **System Console > Authentication > SAML**, paste metadata URL in the **Identity Provider Metadata URL** field, and then select **Get SAML Metadata from IdP**.
+
+This populates the **SAML SSO URL** and the **Identity Provider Issuer URL** fields automatically and the Identity Provider Public Certificate is also downloaded from the server and set locally.
+
+Alternatively you can enter the following fields manually:
```
Comment (catalintomai) [inline source/deployment/sso-saml-adfs-msws2016.rst]: Only the last three fields (from the 6 enumerated) are being impacted by the user entering the metadata Url). The first three can be selected for both alternatives. So I would move the "Alternatively..." statement on top of the last three fields (SSO, IdpUrl and cert) only.

Comment (justinegeffen) [reply source/deployment/sso-saml-adfs-msws2016.rst]: Thanks! Updated it.

**Pull request merged**